runtime/src/accounts_db.rs

```diff
@@ -2878,33 +2878,20 @@ impl AccountsDb {
     pub fn hash_stored_account(
         slot: Slot,
         account: &StoredAccountMeta,
-        cluster_type: &ClusterType,
+        _cluster_type: &ClusterType,
```

Comment:
just making sure: you can stop passing this altogether from the caller (assuming we don't have any immediate plan to change the hashing.).

Reply:
yep, phase 2. ;-) roughing out this is what you meant.